src/pages/instances/InstanceSnapshots.tsx

```diff
@@ -178,15 +178,15 @@ const InstanceSnapshots: FC<Props> = ({ instance }) => {
         },
       ],
       sortData: {
-        name: snapshot.name,
+        name: snapshot.name.toLowerCase(),
```

Comment:
We don't need this anymore, right? The table cannot be sorted by name.

Reply:
true, good catch.